kfs-web/src/main/webapp/WEB-INF/tags/module/purap/puritems.tag

```diff
@@ -379,7 +379,7 @@
 						    attributeEntry="${itemAttributes.itemTypeCode}"
 						    property="document.item[${ctr}].itemTypeCode"
 						    extraReadOnlyProperty="document.item[${ctr}].itemType.itemTypeDescription"
-						    readOnly="${not ( (fullEntryMode and !amendmentEntry) or (amendmentEntry and itemLine.versionNumber == null)) or itemLine.itemTypeCode == 'UNOR' or lockB2BEntry}" tabindexOverride="${tabindexOverrideBase + 0}"/>

```

Comment:
Quantity Taxable (default) and Quantity Tax Exempt from Item Types need to be available on B2B generated Requisitions (bottom of page 5 on the requirements doc). Removing lockB2BEntry makes Item Types editable